cmd/litpy/test_lit.py

```diff
@@ -17,7 +19,8 @@ def __init__(self, i):
 
         # Write a hexkey to the hexkey file
         with open(self.data_dir + "/testkey.hex", 'w+') as f:
-            f.write("1" * 64 + "\n")
+            randomString = ''.join(random.choice("abcdef"+digits) for i in range(64))
```

Comment:
Ah good point. Makes sense that this needs to be different for each node.

In my opinion, we'd still want this to be deterministic instead of random though - perhaps the hex value of the node id. Generally determinism is useful in functional tests for repeatability.